### PR TITLE
add enable googleapis

### PR DESCRIPTION
**Pull request by umeshkumhar**
- Add googleapis for all applications.

### CODE REVIEW

File: applications/jupyter/main.tf [hunk 1]
```diff
@@ -25,6 +25,32 @@ data "google_project" "project" {
   project_id = var.project_id
 }
 
+## Enable Required GCP Project Services APIs
+module "project-services" {
+  source  = "terraform-google-modules/project-factory/google//modules/project_services"
+  version = "~> 14.5"
+
+  project_id = var.project_id
+  activate_apis = flatten([
```
Comment (blackzlq) [inline applications/jupyter/main.tf:34]: dlp?

Comment (umeshkumhar) [reply applications/jupyter/main.tf:34]: fixed.

**Review by imreddy13 — APPROVED** [review]

**Pull request merged**